Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -1783,6 +1783,24 @@ void SoftwareGLContext::gl_fogf(GLenum pname, GLfloat param)
     m_rasterizer.set_options(options);
 }
 
+void SoftwareGLContext::gl_fogi(GLenum pname, GLint param)
+{
+    RETURN_WITH_ERROR_IF(m_in_draw_state, GL_INVALID_OPERATION);
+    RETURN_WITH_ERROR_IF(!(param == GL_EXP || param == GL_EXP2), GL_INVALID_ENUM);
```

Comment:
The next commit claims that we support `GL_LINEAR`, but this check only allows `GL_EXP` and `GL_EXP2` :thinking:

Reply:
Whoops, forgot I left that out... I'll add it in to that commit